server/src/Scoreboard.ts

```diff
@@ -123,16 +109,16 @@ export class Scoreboard {
 
         switch (playerIndex) {
             case PlayerColor.Orange:
-                this._orangeScore += value;
+                this.scoreMap[PlayerColor.Orange].points += value;
                 break;
             case PlayerColor.Green:
-                this._greenScore += value;
+                this.scoreMap[PlayerColor.Green].points += value;
                 break;
             case PlayerColor.Pink:
-                this._pinkScore += value;
+                this.scoreMap[PlayerColor.Pink].points += value;
                 break;
             case PlayerColor.Blue:
-                this._blueScore += value;
+                this.scoreMap[PlayerColor.Blue].points += value;
                 break;
         }
 
```

Comment:
is it a good idea to directly index into `this.scoreMap` with `playerIndex` instead of using switch cases?
```ts
if (playerIndex >= 0 && playerIndex <= 3) {
    this.scoreMap[playerIndex].points += value;
}
```

same question for decrementScore function